support/ci/deploy_mac_launcher.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+# fail fast if we aren't on the desired branch or if this is a pull request
+if  [[ "${TRAVIS_BRANCH}" != "$(cat VERSION)" && ("${TRAVIS_PULL_REQUEST}" != "false" || "${TRAVIS_BRANCH}" != "master") ]]; then
+    echo "We only publish on successful builds of master."
+    exit 0
+fi
+
+BINTRAY_REPO=unstable
+if [ "$(cat VERSION)" == "$TRAVIS_TAG" ]; then
+  BINTRAY_REPO=stable
+fi
+
+# kick off the mac unstable build
+echo "Kicking off the $BINTRAY_REPO mac build"
+var_file=/tmp/our-awesome-vars
+mac_builder=admin@74.80.245.236
```

Comment:
This kind of "magic variable" would be nice to call attention to somehow. One pattern I kinda like is taking all such variables that one might conceivably need to change in the future (i.e., "global" variables, and not just ones that are incidental to the running of the program, such as with the suggestion above to extract `$(cat version)`) and setting them apart in a prominent section of their own (as much as is possible, at least). The idea is that someone reading the script (especially for the first time) has a clear idea of where the points of control are.